### ISSUE TITLE
Spring template

### ISSUE THREAD

**Issue by vithushanms** [serverless/examples#619]
I think we need to have a spring boot template since lot of java developers are interested on this

**Comment by pgrzesik** [comment]
Hello @vithushanms - would you be interested in creating such an example?

**Comment by vithushanms** [comment]
Yes, I have been using few templates in my projects. I would like to add that example to this repo

**Comment by pgrzesik** [comment]
Thanks @vithushanms - we're looking forward to the PR :+1:
